From 9922fe17ad48b5298ad86acdf5b4d850f99e2930 Mon Sep 17 00:00:00 2001
From: Ahmed Salem <aieisalem17@gmail.com>
Date: Wed, 23 Aug 2023 11:59:55 +0100
Subject: Add transparency fix for borders when compositor is running

---
 drw.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drw.c b/drw.c
index a58a2b4..018d862 100644
--- a/drw.c
+++ b/drw.c
@@ -190,6 +190,8 @@ drw_clr_create(Drw *drw, Clr *dest, const char *clrname)
 	                       DefaultColormap(drw->dpy, drw->screen),
 	                       clrname, dest))
 		die("error, cannot allocate color '%s'", clrname);
+
+        dest->pixel |= 0xff << 24;
 }
 
 /* Wrapper to create color schemes. The caller has to call free(3) on the
-- 
2.41.0

